specs/slider/README.md

```diff
@@ -106,13 +119,13 @@ Comments to example 2
 
 ### Subcomponents (pseudo elements)
 
-| selector   | description                              | type                                     |
-| ---------- | ---------------------------------------- | ---------------------------------------- |
-| ::handle   | Allows you to style the handle of the slider | HTML Element. This subcomponent has no subcomponents of its own* |
-| ::bar      | Allows you to style the bar of the slider | HTML Element. This subcomponent has no subcomponents of its own* |
-| ::progress | Allows you to style the progress part of the bar | HTML Element. This subcomponent has no subcomponents of its own* |
+| selector   | description                              |
+| ---------- | ---------------------------------------- |
+| ::handle   | Allows you to style the handle of the slider |
+| ::bar      | Allows you to style the bar of the slider |
+| ::progress | Allows you to style the progress bar of the slider |
+
```

Comment:
Can you give an example for what we might need ::marks? The only thing that I can think of is color. Am I missing smth?